pkg/controller/worker/machine_dependencies.go

```diff
@@ -14,12 +14,281 @@
 
 package worker
 
-import "context"
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/gardener/gardener-extension-provider-packet/pkg/packet"
+	packetclient "github.com/gardener/gardener-extension-provider-packet/pkg/packet/client"
+	util "github.com/gardener/gardener/extensions/pkg/util"
+	v1beta1constants "github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
+	"github.com/pkg/errors"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	vpnSeedConfigMap                      = "equinix-vpn-seed-config"
```

Comment:
```suggestion
      vpnSeedConfigMap = "gardener-extension-provider-equinixmetal-vpn-seed-config"
```

optional: (for consistency/to make clear what this is)

Reply:
Done. Good idea.